data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -240,55 +242,55 @@
          functions see <xref target="PREOF"/>.
        </t>
        <t>
-         The method of instantiating each of the layers is specific to the
-         particular DetNet data plane method. There may be more than one approach
-         that is applicable to a given bearer network type.
+         The method of instantiating each of the layers is specific to the 
+		 particular DetNet data plane method, and more than one approach may 
+		 be applicable to a given bearer network type.
        </t>
 
       <section title="Data Plane Characteristics" anchor="sec_dp_char">
         <t>
-	There are two major characteristics to the data plane:
+	There are two major characteristics to the data plane: the technology and 
+	the encapsulation, as discussed below.
         </t>
+		<section title="Data Plane Technology" anchor="sec_dp_tech">
          <t>
-         <list style='numbers'>
-         <t>
-         Data plane technology: The DetNet data plane is provided by the DetNet 
-		 service and forwarding sub layers.  The DetNet service sub-layer
+         The DetNet service sub-layer
 	 generally provides its functions for the DetNet application flows by using or applying
 	 existing standardized headers and/or encapsulations.  The Detnet
 	 forwarding sub-layer may provide capabilities leveraging that same
-	 header or encapsulation technology e.g. <xref target="dn_svcs"/> or it
-	 may be achieved by other technologies e.g.  <xref
-		 target="dn_svc_encaps"/>.
+	 header or encapsulation technology (e.g., DN IP or DN MPLS) or it
+	 may be achieved by other technologies (e.g.,  <xref
+		 target="dn_svc_encaps"/>).
          DetNet is currently defined for operation over packet switched (IP)
           networks or label switched (MPLS) networks.  
          </t>
+		</section>
+		<section title="Data Plane Format" anchor="sec_dp_form">
          <t>
-          Encapsulation format: DetNet encodes specific flow attributes
-	  (namely flow identity and sequence number) in packets. 
+          DetNet encodes specific flow attributes
+	  (flow identity and sequence number) in packets. 
 	  For example, in DetNet IP, 
           zero encapsulation may be used and no sequence number
           is available, and in DetNet MPLS, DetNet specific information may be added
           explicitly to the packets in the format of S-label and d-CW.  
          </t>
-         </list>
-         </t>
-
-        <figure anchor="dn_svcs" align="center"
-        title="DetNet Services">
-        <artwork align="center"><![CDATA[
+		</section>
 
+     </section>
 
-  +-------+  +---------+
-  | DN IP |  | DN MPLS |
-  +-------+  +---------+
+      <section title="Encapsulation" anchor="sec_dp_encap">
+         <t>
+		 The encapsulation of a DetNet flow allows it to be sent over a   
+		 data plane technology other than its native type. For example, 
+		 an Ethernet TSN app flow can be sent as a DetNet app flow over MPLS.

```

Comment:
This sentence isn't quit right for IP.  How about:

DetNet uses header information to perform traffic classification, i.e., identify DetNet flows, and provide DetNet service and forwarding functions.  As mentioned above, DetNet may add headers, as is the case for DN MPLS, or may headers that are already present, as is the case in DN IP.

Reply:
OK. Agree with proposed change.

Reply:
Typo at "or may headers", I assume s/b "or may use headers".